src/api/spec/controllers/webui/project_controller_spec.rb

```diff
@@ -687,4 +691,34 @@
       is_expected.to redirect_to(project_show_path(user.home_project))
     end
   end
+
+  describe 'POST #new_incident_request' do
+    before do
+      login user
+      request.env["HTTP_REFERER"] = root_url # Needed for the redirect_to :back
+    end
+
+    it "without an existent project will raise an exception" do
+      expect { post :new_incident_request, project: 'non:existent:project' }.to raise_error Project::UnknownObjectError
```

Comment:
Are you sure that this won't raise a 500 because we catch several times ``Project::UnknownObjectError``